private/custom-write.scrbl

```diff
@@ -32,12 +33,12 @@ must satisfy the @racket[custom-write-function/c] contract.
 
 @defthing[custom-write-function/c chaperone-contract?
           #:value (-> any/c output-port? custom-write-mode/c void?)]{
- A @tech{contract} describing functions suitable for use with @racket[
+ A @tech/reference[#:key "Contract"]{contract} describing functions suitable for use with @racket[
```

Comment:
Can the contract `deftech` in the racket reference be adjusted so that we don't need to supply `#:key "Contract"` here?

Reply:
Turns out `contract` works :)